### DISCUSSION TITLE
Need Help: GitHub Education Benefits Not Activated After Acceptance

### DISCUSSION THREAD

**Discussion by yours7himanshu** [community/community#147618] [GitHub Education]
### Hi everyone,

Hello GitHub Community,

I hope you're all doing well. I'm reaching out to seek help regarding an issue with my GitHub Education benefits.

My application for GitHub Education benefits was accepted on 18th December. However, even after two days, my benefits have not yet been activated.

Has anyone else experienced a similar issue? If so, how was it resolved? Is there any additional step I need to take to ensure the activation of my benefits?

I would greatly appreciate any guidance or support from the community on this matter.

Thank you in advance for your help!

Best regards,
Himanshu Dinkar
![Screenshot 2024-12-20 201948](https://github.com/user-attachments/assets/efddce31-fadf-4951-83ec-78e66c4a6dd8)

**Comment by leongrphc** [comment]
im waiting like you i accepted on december 17

**Reply by yours7himanshu** [reply]
may be they are getting lots of github student developer request..Lets see when they will activate that.I will edit response when mine devloper pack will get activated

**Comment by davevad93** [comment]
Hi @yours7himanshu ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @yours7himanshu was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by yours7himanshu** [reply]
Okay i am waiting i will edit my response once my github education developer benifits pack will get activated.
